app/controllers/registrations_controller.rb

```diff
@@ -98,7 +98,6 @@ def render_failure(error)
 
   def form_create
     @registration_form = RegistrationForm.new(nil)
-    create_params = params.require(:registration).to_unsafe_h.merge(current_user:)
```

Comment:
The value of this local var is identical to that of the return value of the `create_params` method, so I suspect this was extracted at some point and we missed this line.

Reply:
The values are identical 🤷🏻‍♂️